src/appellate/utils.js

```diff
@@ -247,4 +262,43 @@ let APPELLATE = {
     }
     return r;
   },
+
+  // Returns an object with the court Id and docket number core extracted from a link to district court
```

Comment:
Helpful to have an example URL here.

Reply:
We have one! I'm going to add an example. Thanks